girder/models/model_base.py

```diff
@@ -901,15 +1041,65 @@ def setUserAccess(self, doc, user, level, save=False):
         :param user: The user to grant or remove access to.
         :type user: dict
         :param level: What level of access the user should have. Set to None
-                      to remove all access for this user.
+            to remove all access for this user.
         :type level: AccessType or None
         :param save: Whether to save the object to the database afterward.
-                     Set this to False if you want to wait to save the
-                     document for performance reasons.
+            Set this to False if you want to wait to save the document for performance reasons.
         :type save: bool
+        :param flags: List of access flags to grant to the group.
+        :type flags: specific flag identifier, or a list/tuple/set of them
+        :param currentUser: The user performing this action. Only required if attempting
+            to set admin-only flags on the resource.
+        :param force: Set this to True to set the flags regardless of the passed in
+            user's permissions (only matters if flags are passed).
```

Comment:
Change this docstring `user's` => `currentUser's`, because there is also a `user` param.